P2946-throws_nothing/P2946.md

```diff
@@ -397,7 +409,7 @@ One use of `[[does_not_throw]]` is to allow defensive checks to throw an
 exception through an otherwise-nonthrowing interface. One proposed way to
 achieve this behavior for non-throwing functions is to use `noexcept` in such a
 way that it can be turned off when desired.  This approach can be implemented
-with help of the preprocessor. For example, using the framework described in
+with the help of the preprocessor. For example, using the framework described in
```

Comment:
"with help from the processor" or "with the help of the processor" or "with the processor's help"